src/main/kotlin/org/rust/ide/refactoring/extractFunction/ui.kt

```diff
@@ -67,23 +70,30 @@ private class DialogExtractFunctionUi(
             updateConfig(config, functionNameField, visibilityBox)
             signatureComponent.setSignature(config.signature)
         }
-        functionNameField.addDataChangedListener {
-            updateConfig(config, functionNameField, visibilityBox)
+
+        val validator = LanguageNamesValidation.INSTANCE.forLanguage(RsLanguage)
```

Comment:
I used that before, but `isValidRustVariableIdentifier` allows names starting with `'`, and that will crash the refactoring, since it's not a valid name for parameters/functions.

Reply:
`isValidRustVariableIdentifier` does not allow such names (in contrast to `isIdentifier`)

Reply:
Ooh, you're right of course. Thank you :)